regtest/docker-compose.yml

```diff
@@ -82,6 +83,32 @@ services:
       - "--bitcoin.zmqpubrawblock=tcp://bitcoind:28332"
       - "--bitcoin.zmqpubrawtx=tcp://bitcoind:28333"
 
+  aperture:
+    build:
+      context: https://github.com/lightninglabs/aperture.git
+      dockerfile: Dockerfile
+      args:
+        checkout: v0.3.8-beta
+    container_name: aperture
+    restart: unless-stopped
+    depends_on:
+      - loopserver
+      - etcd
+    networks:
+      regtest:
+        aliases:
+          - aperture
+    volumes:
+      - "lndserver:/root/.lnd"
+      - "loopserver:/root/loopserver"
+      - "aperture:/root/.aperture"
+    ports:
+      - "11018:11018"
+    entrypoint: [ "/bin/aperture" ]
+    command:
+      - "--configfile=/root/.aperture/aperture.yaml"
+
+
```

Comment:
Could we also bump the client to 0.18.5 pls.

Reply:
Thank you, I updated it.